CITATION.cff

```diff
@@ -0,0 +1,15 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Saksham 
+    given-names: Sharda
+  - family-names: Alessio Paolo 
+    given-names: Buccino
+    orcid: https://orcid.org/0000-0003-3661-527X
+  - family-names: Ben 
+    given-names: Dichter
```

Comment:
"family name" means last name and "given name" means first name. Also, for these types of things use Benjamin as my first name